rustler_mix/lib/mix/tasks/compile.rustler.ex

```diff
@@ -26,6 +26,7 @@ defmodule Mix.Tasks.Compile.Rustler do
   def compile_crate({name, config}) do
     crate_path = Keyword.get(config, :path, "native/#{name}")
     build_mode = Keyword.get(config, :mode, :release)
+    target = Keyword.get(config, :target, nil)
```

Comment:
This new option should possibly be mentioned in the `Crate configuration` section of `rustler_mix`.